wgpu-types/src/lib.rs

```diff
@@ -952,6 +952,19 @@ bitflags::bitflags! {
         ///
         /// This is a native only feature.
         const SHADER_INT64_ATOMIC_ALL_OPS = 1 << 61;
+        /// Allows using the [VK_GOOGLE_display_timing] Vulkan extension.
+        /// This is used for frame pacing to reduce latency, and is generally only available on Android.
```

Comment:
Note that this will be formatted as one paragraph; I recommend to reflow the sentences.